### PR TITLE
Fix dependencies

### PR DESCRIPTION
**Pull request by innovate-invent**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Related issue: https://github.com/JuliaLang/julia/issues/7354

### CODE REVIEW

**Comment by innovate-invent** [comment]
Can the OSX build failure be ignored? There is a different issue that this PR does not fix.
It would be good to get linux support back up and running.

**Comment by wolfv** [comment]
I'll give OS X a try on Monday. If we can't fix it then, we can surely merge Linux for now.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -41,6 +41,7 @@ requirements:
     - arpack
     - suitesparse
     - pcre2 10.23
+    - pcre 8.44
```
Comment (isuruf) [inline recipe/meta.yaml:44]: Why do you need both `pcre` and `pcre2`?

Comment (innovate-invent) [reply recipe/meta.yaml:44]: I am not sure, we could try removing pcre2

Comment (innovate-invent) [reply recipe/meta.yaml:44]: both libraries are required

Comment (isuruf) [reply recipe/meta.yaml:44]: No, they are not. See https://github.com/JuliaLang/julia/blob/731126df8861b97382862617acc1028f22254c15/deps/Versions.make#L3

Comment (innovate-invent) [reply recipe/meta.yaml:44]: I just tried removing pcre2 and the build failed.

Comment (isuruf) [reply recipe/meta.yaml:44]: You need to remove `pcre` and fix the version of `pcre2`.

Comment (innovate-invent) [reply recipe/meta.yaml:44]: the build is specifically looking for pcre as well as pcre2: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=178689&view=logs&j=1bf226d3-0e2f-52d8-fa93-7d9e633347b3&t=ec8d466d-e3b4-5115-4f06-222398f91e6c&l=912

Comment (isuruf) [reply recipe/meta.yaml:44]: That line is about a julia file. Not about `pcre` vs `pcre2`. Can you please try removing `pcre` and fixing the `pcre2` version?

**Comment by innovate-invent** [comment]
@wolfv did you have any luck with OSX? I have several projects depending on fixing this repo. Is it possible to merge this regardless?

**Comment by isuruf** [comment]
For the osx failure, maybe a rerender might fix it.

**Comment by wolfv** [comment]
we might also want to directly work on julia 1.4.2?

**Comment by innovate-invent** [comment]
@conda-forge-admin, please rerender

**Comment by innovate-invent** [comment]
@conda-forge-admin, please rerender

**Comment by innovate-invent** [comment]
Reopening PR due to bug in github losing "Allow edits from maintainers"

**Comment by innovate-invent** [comment]
Reopening PR did not provide option to allow maintainers access. How to I manually rerender?

**Comment by wolfv** [comment]
You can install conda-smithy from conda-forge and run ‘conda smithy rerender’

**Comment by wolfv** [comment]
what's the status here? Do you need support?

**Comment by innovate-invent** [comment]
@wolfv very much so, I have tried turning on the build options that cause it to fetch all of its own dependencies rather than using condas. Even with it managing its own dependencies, it still fails. I think the invocation of the tests needs to be re-approached. I tried switching to `make test` but that doesn't work either. There is a good chance that the entire build.sh needs to be redone as there seems to be significant changes to its build process that are not being captured.

**Pull request closed without merging**